deps/wazuh_testing/wazuh_testing/tools/time.py

```diff
@@ -201,3 +201,62 @@ def time_to_timedelta(time_):
         return timedelta(days=time_value)
     elif time_unit == "w":
         return timedelta(weeks=time_value)
+
+def time_to_human_readable(time_):
```

Comment:
Add two white lines between each function. This applies also at line 233.

Reply:
Of course https://github.com/wazuh/wazuh-qa/pull/686/commits/306de9ae9263f93ba8dd1525933cec0c326bd9b0